apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -156,7 +200,11 @@ const DebugConsole = connect(
       <div
         id="debug-console"
         className={classes}
-        style={this.props.style}
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          ...this.props.style
```

Comment:
Have you tested this in IE11?  I _think_ this will be fine, but there's a [set of known issues with IE11](http://caniuse.com/#feat=flexbox) and `flex-direction: column` that we need to look out for, especially when mixing other style properties in as we do here.